File: java programs/control statements loops/primenumber.java
public class primenumber {
    public static void main(String args[]) {
        int a = 5;
        boolean flag = false;
        for (int i = 2; i < a; ++i) {
            if (a % i == 0) {
                flag = true;
                break;
            }
        }
        if (!flag)
            System.out.println(a + " is a prime nuber");
        else
            System.out.println(a + " is not a prime number");
    }
}

File: PYTHON PROGRAMS/primenumber.py
n = int(input("enter the number: "))
flag= False

if n > 1:
    for i in range(2,n):
        if(n % i) == 0:
            flag = True
            break

if flag ==True:
    print(n," is not a prime number")
else:
    print(n," is a prime number")